src/main/java/helpers/ElementHelper.java

```diff
@@ -10,13 +10,17 @@ public static void scrollToView(WebElement element) {
         js.executeScript("arguments[0].scrollIntoView();", element);
     }
 
-    public static void selectDropdownByText(WebElement element, String text){
+    public static void selectDropdownByText(WebElement element, String text) {
         Select dropdown = new Select(element);
         dropdown.selectByVisibleText(text);
     }
 
-    public static void selectDropdownByIndex(WebElement element, Integer index){
+    public static void selectDropdownByIndex(WebElement element, Integer index) {
         Select dropdown = new Select(element);
         dropdown.selectByIndex(index);
     }
+
+    public static boolean isDisplayed(WebElement element) {
+        return element.isDisplayed();
```

Comment:
this will throw an exception if the element doesn't exist. I mention this many times, not sure why you did get it.

Reply:
Sorry, I I did not pay close attention to this. I defined a new function for check element exists with element size is different with 0. Please review it again, thank you so much!

Reply:
well, what do you really want to validate? validate that the tabs display or just exist?
what happens if the tabs exist in the DOM but not display in the UI?

Reply:
I wanna validate the tabs are displayed and I just updated new code.